### PR TITLE
Copyright of scss files

### PR DESCRIPTION
**Pull request by skjnldsv**
Copyright restauration based on all commits of previous files.

For reference, script I have been using: http://fiddle.jshell.net/sHm3k/40/show/light/

### CODE REVIEW

File: core/css/header.scss [hunk 1]
```diff
@@ -1,8 +1,15 @@
...
- * @copyright Copyright (c) 2017, John Molakvoæ (skjnldsv@protonmail.com)
- * @copyright Copyright (c) 2016, Lukas Reschke (lukas@statuscode.ch)
- * @copyright Copyright (c) 2016, Julius Härtl (jus@bitgrid.net)
- * @copyright Copyright (c) 2016, Jos Poortvliet (jospoortvliet@gmail.com)
+ * @copyright Copyright (c) 2016, John Molakvoæ <skjnldsv@protonmail.com>
+ * @copyright Copyright (c) 2016, Julius Haertl <jus@bitgrid.net>
+ * @copyright Copyright (c) 2016, Lukas Reschke <lukas@statuscode.ch>
+ * @copyright Copyright (c) 2016, Jos Poortvliet <jos@opensuse.org>
+ * @copyright Copyright (c) 2016, Thomas Müller <deepdiver1975@users.noreply.github.com>
+ * @copyright Copyright (c) 2016, Christoph Wurst <christophwurst@users.noreply.github.com>
```
Comment (ChristophWurst) [inline core/css/header.scss]: I think username@users.noreply.github.com comes from merge commits. We should remove those lines/authors.

**Review by ChristophWurst — DISMISSED** [review]

File: core/css/header.scss (hunk 1 above)
Comment (skjnldsv) [inline core/css/header.scss]: Oooh, does that means all those copyrights are wrong? 😞

Comment (ChristophWurst) [inline core/css/header.scss]: Yes, those that end with ``@users.noreply.github.com``

Comment (skjnldsv) [inline core/css/header.scss]: I checked and sometimes it's noreply..github and sometimes not :)
capture d ecran_2017-01-25_16-08-03
capture d ecran_2017-01-25_16-08-10

Comment (ChristophWurst) [inline core/css/header.scss]: just remove the ones that end with github.com. The others are fine :-)

**Review by ChristophWurst — APPROVED** [review]
looks good

**Comment by MorrisJobke** [comment]
> For reference, script I have been using: http://fiddle.jshell.net/sHm3k/31/show/light/

I like this :) Could we add this to our CI chain? So this is then done automatically? :)

File: core/css/styles.scss [hunk 2]
```diff
@@ -1,6 +1,17 @@
...
+/**
+ * @copyright Copyright (c) 2016, John Molakvoæ <skjnldsv@protonmail.com>
+ * @copyright Copyright (c) 2016, Lukas Reschke <lukas@statuscode.ch>
+ * @copyright Copyright (c) 2016, Robin Appelman <robin@icewind.nl>
+ * @copyright Copyright (c) 2016, Julius Haertl <jus@bitgrid.net>
+ * @copyright Copyright (c) 2016, Joas Schilling <coding@schilljs.com>
+ * @copyright Copyright (c) 2016, Morris Jobke <hey@morrisjobke.de>
+ * @copyright Copyright (c) 2016, Christoph Wurst <christoph@winzerhof-wurst.at>
+ * @copyright Copyright (c) 2016, Jan-Christoph Borchardt <hey@jancborchardt.net>
+ * @copyright Copyright (c) 2016, Raghu Nayyar <hey@raghunayyar.com>
```
Comment (jancborchardt) [inline core/css/styles.scss]: This is definitely wrong as the original copyright is 2011, and the file is from then :)

File: core/css/share.scss [hunk 3]
```diff
@@ -1,6 +1,14 @@
 /**
- * @copyright Copyright (c) 2011, Jan-Christoph Borchardt <hey@jancborchardt.net>
- * @copyright Copyright (c) 2017, John Molakvoæ (skjnldsv@protonmail.com)
+ * @copyright Copyright (c) 2016, John Molakvoæ <skjnldsv@protonmail.com>
+ * @copyright Copyright (c) 2016, Morris Jobke <hey@morrisjobke.de>
+ * @copyright Copyright (c) 2016, Julia Bode <julia.bode@lulisaur.us>
+ * @copyright Copyright (c) 2016, Christoph Wurst <christoph@winzerhof-wurst.at>
+ * @copyright Copyright (c) 2015, Hendrik Leppelsack <hendrik@leppelsack.de>
+ * @copyright Copyright (c) 2015, Jan-Christoph Borchardt <hey@jancborchardt.net>
```
Comment (jancborchardt) [inline core/css/share.scss]: Also here, original copyright is 2011

File: core/css/multiselect.scss [hunk 4]
```diff
@@ -1,5 +1,16 @@
...
+ * @copyright Copyright (c) 2016, Sergio Bertolín <sbertolin@solidgear.es>
+ * @copyright Copyright (c) 2015, Hendrik Leppelsack <hendrik@leppelsack.de>
+ * @copyright Copyright (c) 2014, Vincent Petry <pvince81@owncloud.com>
+ * @copyright Copyright (c) 2013, Vincent Petry <pvince81@owncloud.com>
+ * @copyright Copyright (c) 2013, raghunayyar <me@iraghu.com>
+ * @copyright Copyright (c) 2013, Victor Dubiniuk <victor.dubiniuk@gmail.com>
+ * @copyright Copyright (c) 2013, kondou <kondou@ts.unde.re>
+ * @copyright Copyright (c) 2012, Thomas Tanghus <thomas@tanghus.net>
+ * @copyright Copyright (c) 2012, Lukas Reschke <lukas@statuscode.ch>
+ * @copyright Copyright (c) 2012, Jan-Christoph Borchardt <hey@jancborchardt.net>
```
Comment (jancborchardt) [inline core/css/multiselect.scss]: Same here, 2011

File: core/css/apps.scss [hunk 5]
```diff
@@ -1,6 +1,16 @@
...
+ * @copyright Copyright (c) 2016, John Molakvoæ <skjnldsv@protonmail.com>
+ * @copyright Copyright (c) 2016, Julius Haertl <jus@bitgrid.net>
+ * @copyright Copyright (c) 2016, Morris Jobke <hey@morrisjobke.de>
+ * @copyright Copyright (c) 2016, pgys <info@pexlab.space>
+ * @copyright Copyright (c) 2016, Lukas Reschke <lukas@statuscode.ch>
+ * @copyright Copyright (c) 2016, Stefan Weil <sw@weilnetz.de>
+ * @copyright Copyright (c) 2016, Roeland Jago Douma <rullzer@owncloud.com>
+ * @copyright Copyright (c) 2016, jowi <sjw@gmx.ch>
+ * @copyright Copyright (c) 2015, Hendrik Leppelsack <hendrik@leppelsack.de>
+ * @copyright Copyright (c) 2015, Jan-Christoph Borchardt <hey@jancborchardt.net>
```
Comment (jancborchardt) [inline core/css/apps.scss]: 2014, see above in the original line ;)

**Review by jancborchardt — COMMENTED** [review]
Some of the years are definitely wrong ;) @skjnldsv

**Comment by skjnldsv** [comment]
Hum, strange indeed! Let me take a look at what the script returns.

**Comment by skjnldsv** [comment]
@jancborchardt @MorrisJobke 
Okay, There was multiple pages on the api return. :)
Should I keep the oldest date of copyright only? Or should I display all of them?

Example, apps.css
```
 * @copyright Copyright (c) 2017, Roeland Jago Douma <roeland@famdouma.nl>
 * @copyright Copyright (c) 2016, John Molakvoæ (skjnldsv) <skjnldsv@protonmail.com>
 * @copyright Copyright (c) 2016, Julius Haertl <jus@bitgrid.net>
 * @copyright Copyright (c) 2016, Morris Jobke <hey@morrisjobke.de>
 * @copyright Copyright (c) 2016, Roeland Jago Douma <roeland@famdouma.nl>
 * @copyright Copyright (c) 2016, Jan-Christoph Borchardt <hey@jancborchardt.net>
 * @copyright Copyright (c) 2016, pgys <info@pexlab.space>
 * @copyright Copyright (c) 2016, Lukas Reschke <lukas@statuscode.ch>
 * @copyright Copyright (c) 2016, Hendrik Leppelsack <hendrik@leppelsack.de>
 * @copyright Copyright (c) 2016, Thomas Müller <deepdiver1975@users.noreply.github.com>
 * @copyright Copyright (c) 2016, Thomas Müller <thomas.mueller@tmit.eu>
 * @copyright Copyright (c) 2016, jowi <sjw@gmx.ch>
 * @copyright Copyright (c) 2015, Morris Jobke <hey@morrisjobke.de>
 * @copyright Copyright (c) 2015, Thomas Müller <thomas.mueller@tmit.eu>
 * @copyright Copyright (c) 2015, Vincent Petry <pvince81@owncloud.com>
 * @copyright Copyright (c) 2015, Jan-Christoph Borchardt <hey@jancborchardt.net>
 * @copyright Copyright (c) 2015, Lukas Reschke <lukas@owncloud.com>
 * @copyright Copyright (c) 2015, fuhaoshih <fuhaoshih@gmail.com>
 * @copyright Copyright (c) 2015, Arthur Schiwon <blizzz@owncloud.com>
 * @copyright Copyright (c) 2015, Joas Schilling <nickvergessen@owncloud.com>
 * @copyright Copyright (c) 2015, Volker E <volker.e@temporaer.net>
 * @copyright Copyright (c) 2015, Bernhard Posselt <dev@bernhard-posselt.com>
 * @copyright Copyright (c) 2014, Jan-Christoph Borchardt <hey@jancborchardt.net>
 * @copyright Copyright (c) 2014, Thomas Müller <thomas.mueller@tmit.eu>
 * @copyright Copyright (c) 2014, Lukas Reschke <lukas@owncloud.com>
 * @copyright Copyright (c) 2014, Morris Jobke <hey@morrisjobke.de>
 * @copyright Copyright (c) 2014, Bernhard Posselt <dev@bernhard-posselt.com>
```

**Comment by skjnldsv** [comment]
I updated the script :)

**Comment by jancborchardt** [comment]
@skjnldsv cool, looks good! :)

**Pull request merged**